src/light.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+#ifndef CATA_SRC_LIGHT_H
+#define CATA_SRC_LIGHT_H
+
+#include <cmath>
+#include <ostream>
+
+
+// Compacter representation
```

Comment:
nitpick: As an English major (albeit non-native), I would expect "more compact representation"

Reply:
Thanks! I wonder how these simple errors creep in because you're obviously right :sweat_smile: